initialization.py

```diff
@@ -1,9 +1,8 @@
 import numpy as np
-import kmeans_functions as kmf
+import scipy.spatial.distance as ssdist
 
 
 def random_init(data, n_clusters):
```

Comment:
This function should operate accepting `random_state`, just like `sklearn` estimators. See [this](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/cluster/_kmeans.py#L843), [this](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841d57e8b5f6b2a570312e9d832e69debc/sklearn/utils/validation.py#L800) and [this](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/cluster/_kmeans.py#L926). Alternatively, you can try [another approach](https://github.com/gmrukwa/divik/blob/master/divik/core/_seed.py#L17).